internal/pkg/aws/cloudformation/changeset_test.go

```diff
@@ -0,0 +1,353 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package cloudformation
+
+import (
+	"errors"
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/cloudformation"
+	"github.com/aws/copilot-cli/internal/pkg/aws/cloudformation/mocks"
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/require"
+)
+
+func TestChangeSet_createAndExecute(t *testing.T) {
```

Comment:
Can we instead capture this part of our exposed APIs like the `UpdateStack` test instead of creating a separate one here

Reply:
addressed~